ddtrace/_trace/tracer.py

```diff
@@ -779,7 +779,9 @@ def _start_span(
             if span._local_root is None:
                 span._local_root = span
             for k, v in _get_metas_to_propagate(context):
-                if k != SAMPLING_DECISION_TRACE_TAG_KEY:
+                from ddtrace.appsec._constants import APPSEC
```

Comment:
probably don't need to import in the for loop?

Reply:
definitely, sorry I was just testing if this approach would work, moving the import to the top!